core/src/main/scala/com/salesforce/op/stages/impl/tuning/DataSplitter.scala

```diff
@@ -46,11 +47,13 @@ case object DataSplitter {
    */
   def apply(
     seed: Long = SplitterParamsDefault.seedDefault,
-    reserveTestFraction: Double = SplitterParamsDefault.ReserveTestFractionDefault
+    reserveTestFraction: Double = SplitterParamsDefault.ReserveTestFractionDefault,
+    maxTrainingSample: Int = SplitterParamsDefault.MaxTrainingSampleDefault
   ): DataSplitter = {
     new DataSplitter()
       .setSeed(seed)
       .setReserveTestFraction(reserveTestFraction)
+      .setMaxTrainingSample(maxTrainingSample)
```

Comment:
is this also exposed for the datacutter class?

Reply:
Yes, I added it to SplitterParams which datacutter has access to - e4b8a9219a73291d9e1dbc22274ef97f46898f25. So that I can use the same set/get functions across DataBalancer, DataCutter and DataSplitter.